apisix/cli/ngx_tpl.lua

```diff
@@ -51,6 +51,8 @@ worker_shutdown_timeout {* worker_shutdown_timeout *};
 env APISIX_PROFILE;
 env PATH; # for searching external plugin runner's binary
 
+env APISIX_BYPASS_ADMIN_API_AUTH; # bypass admin api auth
```

Comment:
```suggestion
env ADMIN_API_TOKEN_REQUIRED = true; # token is required to access the admin API
```

Reply:
Sounds weird...

I prefer the variable names in bitnami/etcd that bypass authentication:

![image](https://user-images.githubusercontent.com/24536972/228484731-4840856c-c0ba-4cfe-bd2e-d669fe565844.png)

`ALLOW_NONE_AUTHENTICATION` looks more natural. 

If we use `ADMIN_API_TOKEN_REQUIRED`, users should bypass authentication by specifying `false`.

IMHO, `false` is not a good value to make users feel positive.

Reply:
Ref: https://hub.docker.com/r/bitnami/etcd/

Reply:
I don't think etcd has a good name for this configuration.
Straightforward, without explanation, is a good name.

Reply:
> IMHO, `false` is not a good value to make users feel positive.

you are right. So we don't suggest the user set it to `false` in prod.